integration/no_package_manager_test.go

```diff
@@ -0,0 +1,113 @@
+package integration_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testNoPackageManager(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		pack   occam.Pack
+		docker occam.Docker
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack()
+		docker = occam.NewDocker()
+	})
+
+	context("when building an app with no package manager", func() {
+		var (
+			image     occam.Image
+			container occam.Container
+
+			name   string
+			source string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
+			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
+			Expect(docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
+			Expect(os.RemoveAll(source)).To(Succeed())
+		})
+
+		it("creates a working OCI image with a start command", func() {
+			var err error
+			source, err = occam.Source(filepath.Join("testdata", "no_package_manager"))
+			Expect(err).NotTo(HaveOccurred())
+
+			var logs fmt.Stringer
+			image, logs, err = pack.WithNoColor().Build.
+				WithBuildpacks(pythonBuildpack).
+				WithPullPolicy("never").
+				Execute(name, source)
+			Expect(err).NotTo(HaveOccurred(), logs.String())
+
+			Expect(logs).To(ContainLines(ContainSubstring("CPython Buildpack")))
+			Expect(logs).To(ContainLines(ContainSubstring("Python Start Buildpack")))
+
+			container, err = docker.Container.Run.
```

Comment:
My only quibble is that I'd expect this test to create an image that launches a repl on run, but the test is overriding the launch cmd.  Could this check for the repl instead?

Reply:
We figured that the `python-start` integration tests would be checking for that start command specifically (and that these tests are more about ensuring that the right order group is activated via build logs).

Reply:
I believe paketo language families do end testing on the resultant image (e.g make sure ca-certs loaded in the final image). Not a strong opinion - ready to merge if you think this is fine.

Reply:
After bumping API version to 0.4, we'll be able to test that the `python-start` sets the right start command *and* that we can append a target file as an arg, which will be nice end-to-end testing.

Reply:
Thanks